lib/elixir/lib/task/supervised.ex

```diff
@@ -168,6 +169,7 @@ defmodule Task.Supervised do
       reducer: reducer,
       monitor_pid: monitor_pid,
       monitor_ref: monitor_ref,
+      ordered: ordered?,
```

Comment:
I am 👍 to using `?` with boolean options but I'm not sure about the rest of the team 😕